### DISCUSSION TITLE
Usando HTTP ao invés de HTTPS

### DISCUSSION THREAD

**Discussion by caiocinel** [BrasilAPI/BrasilAPI#344] [Q&A]
Existe algum modo de realizar solicitações sem SSL? Trabalho com uma plataforma antiga que tem incompatibilidade com HTTPS, seria uma mão na roda se existisse alguma maneira de não usa-lá, e ignorando redirects acabo recebendo um status 401.

**Comment by LorhanSohaky** [comment]
Acho que é a própria vercel que coloca https, não sei se temos controle sobre isso

**Reply by RodriAndreotti** [reply]
Na realidade, o certificado digital é fornecido pela CloudFlare, geralmente existe a possibilidade de desativar o redirecionamento automático, agora precisamos ver se da CloudFlare para a Vercel tem algum ssl obrigatório,
